docs/collections/_schema/human-readable-schema.md

```diff
@@ -0,0 +1,226 @@
+
+```cedar
+{
+  name: String,
+  features: {
+    age: Long,
+    height: Long,
+    eyecolor: String
+  }
+}
+```
+
+#### Set {#schema-entitytypes-shape-set}
+{: .no_toc }
+
+A set type declaration consists of keyword `Set` and an element type surrounded by brackets (`<>`). For example, `Set<Long>` is the type of a set of `Long`s.
+
+
+## Actions {#schema-actions}
+
+The following action declaration specifies an action `ViewDocument`. It is a child action of action group `ReadActions` and applies to principals of entity type `User` and `Public`, resources of entity type `Document` and contexts of record type `{ network: ipaddr, browser: String}`.
+
+```cedar
+action ViewDocument in [ReadActions, ExampleNS::Action::"Write"] appliesTo {
+    principal: [User,Public],
+    resource: Document,
+    context: {
+        network: ipaddr,
+        browser: String
+    }
+};
+```
+
+An action name is either an identifier or a string. The membership relation syntax of action declarations is akin to that of entity declarations. The difference is that parent action names could be strings but entity type names must be identifiers. If a parent action is declared in another namespace, its name must be a *fully-qualified action entity name*. For example, the `Write` action declared in the namespace `ExampleNS` must be referred to as `ExampleNS::Action::"Write"` in the above action declaration.
+
+The `appliesTo` construct specifies an action's applicability. It is a record of three optional keys: `principal`, `resource`, and `context` that specifies principals, resources, and contexts to which the action apply. Absence of the `appliesTo` construct means that the actions do not apply to any principals/resources/contexts. `principal` or `resource` keys, if given, must an entity type or a non-empty list of entity types. Absence of `principal` or `resource` keys means that the action applies to *unspecified* principals or resources, respectively.
+The `context` value must be a record and its absence defaults to an empty record.
+
+## Common types {#schema-commonTypes}
+
+Like the JSON schema format, human-readable schema syntax allows for declarations of common types so that entity type declarations can use them to avoid error-prone duplication. The syntax of common type declarations is similar to defining type aliases in most programming languages: `type <Id> = <Type>` . The right hand side of `=` is a schema type name except for common types to avoid definitional circularity.
+
+## Type name disambiguation
+
+Type names in the human-readable schema format can conflict with each other. For example, `ipaddr` is a valid unqualified common type name as well as an extension type name. `Foo::Bar` is a valid qualified common type name and an entity type name. We use the following rules to disambiguate type names.
+
+1. Primitive and extension type names cannot alias by design.
```

Comment:
This rule is confusing. Is it just saying that we haven't made any extensions that have the same name as a primitive? Given the extension types are practically primitives from the users perspective, this is hardly worth mentioning.

Reply:
I see you approved the PR. Anything need to be done here?

Reply:
I think what it meant is we won't allow new extension type names to alias with primitive types. I agree it's kind of obvious.